Modules/MID/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# ---- Library ----
+
+add_library(QcMIDRaw)
```

Comment:
I would propose to stick to a single library per detector, thus `QcMID` and to have all the classes in there (raw, cluster, you name it). I know that ITS had a raw library but it was the beginning and we did not know better.